.travis.yml

```diff
@@ -59,6 +65,15 @@ before_script:
   - npm install --progress false --quiet || ( cat npm-debug.log && false )
   - bower install --config.interactive=false --quiet
   - cd ../
+
+  # Required for cypress tests; TODO: Move to puppet.
+  # TODO: Start up solr (and neo4j?)
+  - python manage.py loaddata guest.json
+  - python manage.py activate_user guest
+  - python manage.py add_users_to_public_group
+  - pushd ui && grunt make && popd
+  - python manage.py collectstatic --noinput -v 0
+  - python manage.py runserver --insecure --settings config.settings.prod_except_email &
```

Comment:
This configuration doesn't test any actual scenario. In dev mode Django devserver is used with `DEBUG=True`. In prod mode Apache is used with `DEBUG=False`.

Also, there are very few bugs that could appear in prod mode but not in dev and they would be highly dependent on the deployment environment anyway, so I would simply run Cypress tests with dev settings.